src/engineering-meeting/meeting-randomiser.ts

```diff
@@ -16,27 +20,40 @@ export class MeetingRandomiser {
       this.nameRandomiser = nameRandomiser
    }
 
-   randomise(): { name: string, blocks: BuildingBlock[], duration: string, generatedDate: string } {
+   async randomise(): Promise<Content> {
       const name = this.nameRandomiser.randomise()
       const blocks = this.blockRandomiser.randomise()
       const durationMin = blocks.reduce((acc, block) => acc + block.duration.minimum, 0)
       const durationMax = blocks.reduce((acc, block) => acc + block.duration.maximum, 0)
       const duration = `${durationMin} - ${durationMax} minutes`
       const generatedDate = getDateTime()
-      return { name, blocks, duration, generatedDate }
+      try {
+         const result = await fetchWithTimeout(
+            `https://novoda-dreams.loca.lt/dreams?prompt="${name}.${characteristics.join('.')}"`,
+            {
+               headers: { "Bypass-Tunnel-Reminder": "true" },
+            },
+            10_000
+         );
+         const blob = await result.blob();
+         return new Content(blocks, name, duration, generatedDate, URL.createObjectURL(blob));
+      } catch {
+         await delay(2000)
```

Comment:
When the API is not available, and it will fail, we still want to show to loading state for a bit. Poor Roberto otherwise, let's pretend the bot is also doing something on this scenario!